.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Run dev",
+            "request": "launch",
+            "runtimeArgs": [
+                "run-script",
+                "dev"
+            ],
+            "localRoot": "${workspaceFolder}\\InfoLogger\\",
```

Comment:
I see that you have used the `Windows` path for localRoot. This of course will not work for unix based operating systems.
Could you please create a configuration for windows and one for unix based systems in which the forward slash is used for the path?
Ideally, the name of the configuration should also reflect this